pyls/plugins/definition.py

```diff
@@ -6,8 +6,9 @@
 
 
 @hookimpl
-def pyls_definitions(document, position):
-    definitions = document.jedi_script(position).goto_assignments()
+def pyls_definitions(document, position, config):
+    follow_imports = config.settings().get('plugins').get('jedi', {}).get('follow_imports', False)
```

Comment:
`pyls.plugins.jedi` doesn't exist. Do you mean `jedi_definitions`? In which case, you can use `config.plugin_settings('jedi_definitions')`.

Other comment is that settings in pyls are camelCase (conforming to VSCode settings convention) the

Also, whilst you here it would be good to add support for `follow_builtin_imports`